lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -254,6 +254,27 @@ def test_course_display_name_not_override_with_course_title(self):
         self.assertIn('refundable course', response.content)
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_course_number_override(self):
```

Comment:
Or we could rename the test to something like 'test_course_display_overrides' -- that would be acceptable as well